src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -13,6 +13,7 @@ import useRootChannelListHook from '../../hooks/screen/useRootChannelListHook';
 
 const ChannelListScreenV2 = () => {
   const navigation = useNavigation();
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment indicates that `useProfileHook` should be replaced with `useUserAuthHook`. However, this change has not been made in the code. This could lead to potential issues if the `useUserAuthHook` provides different functionality or data than `useProfileHook`.
```diff
-   const {profile} = useProfileHook();
+   const {profile} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->